admin/partials/cover_art.php

```diff
@@ -34,9 +42,21 @@
 				$add_hidden = '';
 				$remove_hidden = 'hidden';
 			} ?>
-        </div>
-        <input name="apple_news_coverart_<?php echo esc_attr( $key ); ?>" class="apple-news-coverart-id" type="hidden" value="<?php echo esc_attr( $image_id ); ?>" />
-        <input type="button" class="button-primary apple-news-coverart-add <?php echo esc_attr( $add_hidden ); ?>" value="<?php echo esc_attr( __( 'Add image', 'apple-news' ) ); ?>" />
-        <input type="button" class="button-primary apple-news-coverart-remove <?php echo esc_attr( $remove_hidden ); ?>" value="<?php echo esc_attr( __( 'Remove image', 'apple-news' ) ); ?>" />
-    </div>
+		</div>
+		<input name="<?php echo esc_attr( $key ); ?>"
+			class="apple-news-coverart-id"
+			type="hidden"
+			value="<?php echo esc_attr( $image_id ); ?>"
+			data-height="<?php echo esc_attr( $data['height'] ); ?>"
+			data-width="<?php echo esc_attr( $data['width'] ); ?>"
+		/>
+		<input type="button"
+			class="button-primary apple-news-coverart-add <?php echo esc_attr( $add_hidden ); ?>"
+			value="<?php echo esc_attr( __( 'Add image', 'apple-news' ) ); ?>"
```

Comment:
Not a blocker, but there are `esc_attr__()` and `esc_attr_e()` functions.

https://codex.wordpress.org/Function_Reference/esc_attr_2
https://codex.wordpress.org/Function_Reference/esc_attr_e